maintenance.html

```diff
@@ -0,0 +1,38 @@
+
+<!DOCTYPE html >
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,maximum-scale=1">
+    <link rel="search" type="application/opensearchdescription+xml" title="Friendica Directory" href="/opensearch" />
+
+    <title>Friendica Directory</title>
+    <link rel="shortcut icon" href="/images/friendica-32.png" />
+    <link rel="apple-touch-icon" href="/images/friendica-128.png"/>
+
+    <link href="/assets/css/style.css" rel="stylesheet" type="text/css" />
+    <link href="/assets/css/awesome/font-awesome.min.css?v=4.3.0" rel="stylesheet" type="text/css">
+
+    <script type="text/javascript" src="/assets/js/jquery/jquery.min.js?v=1.11.2"></script>
+    <script type="text/javascript" src="/assets/js/main.js"></script>
+</head>
+<body>
+<a class="hamburger mobile"><i class="fa fa-bars"></i></a>
+<nav id="links">
+    <div class="viewport">
+    </div>
+</nav>
+<div class="homepage-wrapper">
+    <h1 class="header">
+        Friendica &nbsp; &nbsp;<br>&nbsp; &nbsp; Directory
+    </h1>
+    <h2>
+        Directory under maintenance
+    </h2>
+
+    <p class="about">
+        The Friendica Directory is unavailable at this time, please connect with <a href="https://friendica.mrpetovan.com/profile/hypolite">hypolite@friendica.mrpetovan.com</a> for updates about its status.
```

Comment:
You're right, I committed directly to my master and then I built the two other issues branches on my master as well, which means that the commits with the maintenance page appears everywhere, both in this PR as well as in the branches' PR.

I'm not sure I can make it a setting as it is supposed to be a static page that should work even if nothing else does.